storage/pipeline/commit_batch.go

```diff
@@ -284,43 +282,6 @@ func (b *CommitBatcher) maybeStartBatch(notif bool) ([]sealiface.CommitBatchRes,
 		return nil, err
 	}
 
-	if individual {
```

Comment:
It's not entirely clear to me if we ever get to this path now, post-nv22; I think from the larger block above this that if we're post-nv22 that we always end up with `processBatchV2` (btw the comment in there, `ProveCommitSectors2` should probably be changed to `ProveCommitSectors3`) and then clear out those sectors from the list so even when we land in this branch we don't end up calling `processIndividually()` at all.

i.e. this cleanup is mainly a dead-code removal exercise, isn't it? There shouldn't be active use of PCS post-nv22 with lotus-miner using the existing code paths, correct?

Reply:
There was a recent issue from one of the SP where they go method not found error. They somehow reached PCS1 path code. Removal simply clean up things for good leaving nothing to chance.

Reply:
Oh, so we should also push out a lotus-miner release at some point after this as well? We've separated the node and miner release process so we need to explicitly opt to do this.

Reply:
Yes. We should ship a new miner release for NV23.

Reply:
This path was active when basefee is below the batch balancer level.